client/landing/jetpack-cloud/components/threat-item-subheader/index.tsx

```diff
@@ -44,8 +44,8 @@ const ThreatItemSubheader: React.FC< Props > = ( { threat } ) => {
 			case 'file':
 				return (
 					<>
-						{ translate( 'Threat found %(signature)s', {
-							args: {
+						{ translate( 'Threat found {{signature/}}', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Threat found ({{signature/}})' )` *ES Score: 12*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/threat-item-subheader-on-file-threats)